WordPress/Classes/ViewRelated/Stats/Charts/StatsLineChartView.swift

```diff
@@ -232,36 +241,62 @@ private extension StatsLineChartView {
         let yAxis = leftAxis
 
         yAxis.axisLineColor = styling.lineColor
-        yAxis.axisMinimum = 0.0
         yAxis.drawAxisLineEnabled = false
         yAxis.drawLabelsEnabled = true
         yAxis.drawZeroLineEnabled = true
         yAxis.gridColor = styling.lineColor
         yAxis.labelTextColor = styling.labelColor
         yAxis.labelFont = WPStyleGuide.fontForTextStyle(.footnote, symbolicTraits: [], maximumPointSize: WPStyleGuide.Stats.maximumChartAxisFontPointSize)
-        yAxis.setLabelCount(Constants.verticalAxisLabelCount, force: true)
         yAxis.valueFormatter = styling.yAxisValueFormatter
         yAxis.zeroLineColor = styling.lineColor
 
         // This adjustment is intended to prevent clipping observed with some labels
         // Potentially relevant : https://github.com/danielgindi/Charts/issues/992
         extraTopOffset = Constants.topOffset
-    }
-
-    func configureYAxisMaximum() {
-        let lowestMaxValue = Double(Constants.verticalAxisLabelCount - 1)
 
-        if let maxY = data?.getYMax(axis: .left),
-           maxY >= lowestMaxValue {
-            leftAxis.axisMaximum = VerticalAxisFormatter.roundUpAxisMaximum(maxY)
-        } else {
-            leftAxis.axisMaximum = lowestMaxValue
+        guard let data else { return }
+        let yAxisMax = data.getYMax(axis: .left)
+
+        if statType == .viewsAndVisitors {
+            yAxis.setLabelCount(Constants.verticalAxisLabelCount, force: true)
+
+            yAxis.axisMinimum = 0
+
+            let lowestMaxValue = Double(Constants.verticalAxisLabelCount - 1)
+            let dataYMax = yAxisMax
+            if dataYMax >= lowestMaxValue {
+                yAxis.axisMaximum = VerticalAxisFormatter.roundUpAxisMaximum(dataYMax)
+            } else {
+                leftAxis.axisMaximum = lowestMaxValue
+            }
+        } else if statType == .subscribers {
+            if areDataValuesIdentical {
```

Comment:
```suggestion
            if data.getYMax(axis: .left) == data.getYMin(axis: .left) {
```

Reply:
This might have been an oversight. Thanks for spotting it!

Reply:
Fixed in 0924f99ae5f8aea34cf71f13f6491699fcd10b07! Makes the code much simpler 😅